turbojpeg.py

```diff
@@ -90,13 +111,23 @@
 TJFLAG_STOPONWARNING = 8192
 TJFLAG_PROGRESSIVE = 16384
 
+class CroppingRegion(Structure):
+    _fields_ = [("x", c_int), ("y", c_int), ("w", c_int), ("h", c_int)]
+
+class TransformStruct(Structure):
+    _fields_ = [("r", CroppingRegion), ("op", c_int), ("options", c_int),
+        ("data", c_void_p), ("customFilter", c_int)]
```

Comment:
The `customFilter` field definition isn't correct.  See https://docs.python.org/3/library/ctypes.html#callback-functions for function pointer types.